wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -173,8 +173,12 @@ public static Command defer(Supplier<Command> supplier, Set<Subsystem> requireme
    *
    * @param supplier the command supplier
    * @return the command
+   * @deprecated the ProxyCommand supplier constructor has been deprecated in favor of directly
+   *     proxying a {@link DeferredCommand}, see ProxyCommand documentaion for more details.
    * @see ProxyCommand
    */
+  @Deprecated(since = "2024", forRemoval = true)
+  @SuppressWarnings("removal")
   public static Command deferredProxy(Supplier<Command> supplier) {
```

Comment:
This factory is new for 2024, so it's probably useful.
Change the impl?

Reply:
No, we should just remove it.  There's no reason for it to exist c.f. `defer(proxy(...))``

Reply:
Removing it would break our deprecation policy

Reply:
The healthy policy has occasional exceptions.  The user-side fix for the removal here is trivial and this is confusing clutter.

Reply:
I thought we were following the policy by deprecating for removal instead of just removing?

Reply:
We always deprecate things for at least one year unless it's impossible to do so.